pkg/reconciler/pingsource/controller.go

```diff
@@ -97,5 +111,10 @@ func NewController(
 			)),
 	})
 
+	serviceaccountInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
```

Comment:
could you possible handle this in the existing event handler function? not sure if having two different handlers on the same filtering resource will cause any bugs. @pierDipi do you know if this is okay to do?

Reply:
@Cali0707 which existing event handler? I'm not sure I follow exactly

Reply:
Wait never mind, I got confused while reading the code -  my bad!